.pullapprove.yml

```diff
@@ -1,8 +1,10 @@
 approve_by_comment: true
-approve_regex: ^LGTM
+approve_regex: '^(lgtm|LGTM)'
```

Comment:
I know non-maintainers' lgtm won't count, just don't think it necessary to introduce such flexibility, my 2 cents, I don't object if other people want this.